accounts/templates/partials/_reading_goal.html

```diff
@@ -0,0 +1,142 @@
+{% load static %}
+{% if goal %}
+<!-- Reading Resolution -->
+<section id="reading_goal_section" class="flex flex-col max-md:w-10/12 mx-auto my-10 space-y-12">
+    <div class="flex flex-col bg-[#6f42c166] rounded-3xl">
```

Comment:
I see this color used in other places, should we consider adding it to tailwind's config as a shared color?

Reply:
Good call

Reply:
Addressed in next commit